src/StudioCore/ParamEditor/ParamEditorView.cs

```diff
@@ -72,7 +72,7 @@ private void ParamView_ParamList_Header(bool isActiveView)
         var resAutoParam = AutoFill.ParamSearchBarAutoFill();
         if (resAutoParam != null)
         {
-            _selection.SetCurrentRowSearchString(resAutoParam);
+            _selection.currentParamSearchString = resAutoParam;
```

Comment:
Pretty funny how long this bug must've survived.

Can we add a function to handle this instead of direct assignment?
I have a suspicion we might need to clear some ui cache state as we do for rows. I doubt nav history would get involved here, but keeping the selection/viewstate logic encapsulated is ideal, I think.

Nonetheless, thanks for the catch.

Reply:
I added a method.

Reply:
Cheers. Happy with this if George or Kata are on a merge spree, or if I get around to merging bits myself.